chaos_genius/core/utils/data_loader.py

```diff
@@ -110,13 +143,11 @@ def _get_tz_from_offset_str(self, utc_offset_str="GMT+00:00"):
 
         timezones = pytz.all_timezones
         for tz_name in timezones:
-            try:
+            with contextlib.suppress(AttributeError):
```

Comment:
TIL.

What do you think about using `getattr(tz, "_transition_info", ((None,),))` instead? Suppressing an entire class of errors means that this error will also be ignored if it's produced somewhere down the stack - for example in `pytz.timezone` function.

Reply:
added TODO for this